app/javascript/profileDropdown/flagButton.js

```diff
@@ -58,10 +34,49 @@ export function initFlag() {
             flagButton.innerHTML = `Flag ${profileUserName}`;
           }
         })
-        .catch((e) => window.alert(`Something went wrong: ${e}`));
+        .catch((e) => {
+          Honeybadger.notify(
+            isUserFlagged ? 'Unable to unflag user' : 'Unable to flag user',
+            profileUserId,
+          );
+          window.alert(`Something went wrong: ${e}`);
+        });
```

Comment:
not sure this valid syntax (yes, I can just test this myself)

specifically, can I provide a block here to replace the single method before - or do I need to _also_ call the block (i.e. wrap this in a closure and call it, making the catch a single statement again)?

Reply:
answered my own question - the block is fine in place of a single expression.